src/localisation/language.cpp

```diff
@@ -156,7 +156,7 @@ const char *language_get_string(rct_string_id id)
 
 bool language_open(int id)
 {
-	static const char *languagePath = "%s/language/%s.txt";
+	#define LANGUAGE_PATH  "%s/language/%s.txt"
```

Comment:
Because then compiler can warn you if you try doing something like `printf("%s", 12);`. But this doesn't look so nice with all the warnings raised now and I will check how best to solve it – perhaps lower this warning a little.